### PR TITLE
Add Zyxel cfg lan get --Name name

### PR DESCRIPTION
**Pull request by elavaud**
##### ISSUE TYPE
 - New Template Pull Request

##### COMPONENT
Zyxel OS, cfg lan get --Name <name>

### CODE REVIEW

File: ntc_templates/templates/index [hunk 1]
```diff
@@ -506,3 +506,5 @@ watchguard_firebox_show_arp.textfsm, .*, watchguard_firebox, sh[[ow]] arp
 
 yamaha_show_environment.textfsm, .*, yamaha, sh[[ow]] en[[vironment]]
 yamaha_show_ip_route.textfsm, .*, yamaha, sh[[ow]] ip ro[[ute]]
+
+zyxel_os_cfg_lan_get_--Name_name.textfsm, .*, zyxel_os, cfg lan get --Name .+
```
Comment (itdependsnetworks) [inline ntc_templates/templates/index:510]: Similar to previous, does "cfg lan get" work?

Comment (elavaud) [reply ntc_templates/templates/index:510]: Unfortunately not. Much more information will be provided when fetching a specific lan info

Comment (itdependsnetworks) [reply ntc_templates/templates/index:510]: Let's at least make it `\S+`, presuming there can only be a name that does not include a space

Comment (elavaud) [reply ntc_templates/templates/index:510]: I managed to create one with space... No special characters tho. Maybe just a `[\s\w-]+` ?

**Review by jmcgill298 — COMMENTED** [review]
needs rebase

**Comment by elavaud** [comment]
Grouped in #1142

**Pull request closed without merging**